packages/application-shell/src/components/fetch-project/fetch-project.js

```diff
@@ -145,53 +145,48 @@ export const mapAllDataFencesToGroupedObjectShape = allAppliedDataFences => {
   );
 };
 
-class FetchProject extends React.Component {
-  static displayName = 'FetchProject';
-  static propTypes = {
-    projectKey: PropTypes.string,
-    skip: PropTypes.bool,
-    children: PropTypes.func.isRequired,
-  };
-  static defaultProps = {
-    skip: false,
-  };
-  shouldComponentUpdate(nextProps) {
-    return !deepEqual(this.props, nextProps);
```

Comment:
I couldn't find an useful case for this check, so I left it out when migrating the component to FC.